### PR TITLE
feat(content): Clarify local deployment in "Getting Started"

### PR DESCRIPTION
**Pull request by stormwild**

### CODE REVIEW

**Review by ndneighbor — APPROVED** [review]
D'oh! Good catch.

**Review by ndneighbor — APPROVED** [review]

**Review by half0wl — APPROVED** [review]

**Comment by half0wl** [comment]
@stormwild Thanks for the PR! Apologies for the late response here - must've gotten lost in an ocean of things, sorry 'bout that.

This looks good to go! Could you resolve the conflicts so we can merge this in?

**Comment by stormwild** [comment]
> @stormwild Thanks for the PR! Apologies for the late response here - must've gotten lost in an ocean of things, sorry 'bout that.
> 
> This looks good to go! Could you resolve the conflicts so we can merge this in?

@half0wl PR updated with latest from upstream main

File: src/docs/getting-started.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ Open `src/pages/index.tsx` in a text editor of your choice and change line 79 of
 
 If you are running the dev server locally, you'll see the change. Feel free to change anything within the project as well! We can't stop you.
 
-Once you are done, save the file. In the next step, we will show you how easy it is to deploy your changes.
+Once you are done, save the file, then commit and push to your repo. In the next step, we will show you how easy it is to deploy your changes.
```
Comment (half0wl) [inline src/docs/getting-started.md:155]:
```suggestion
Once you are done, save the file. If you're in a Git repository, you should also commit and push to your repository. In the next step, we will show you how easy it is to deploy your changes.
```

**Review by half0wl — CHANGES_REQUESTED** [review]
@stormwild thank you! Requesting one minor change that I didn't notice until reading this in context, sorry :-)

**Comment by ndneighbor** [comment]
Committed the suggested changes on Alexander's behalf! Merging! Cheers :)

**Review by half0wl — APPROVED** [review]
Thanks @stormwild & @ndneighbor!

**Pull request merged**